### PR TITLE
Lesson 12 fix grammar and completeness

### LINKED ISSUE #68: [Review] 12-visualization-relationships
# This lesson has been reviewed and resolved of the following issues
- [x] Typos
- [x] Grammar errors
- [x] Missing links
- [x] Broken Images
- [x] Checked for completeness
- [x] Quiz (if no quiz assign to paladique)

### PR DESCRIPTION
**Pull request by iofall**
Fix #68

### CODE REVIEW

File: 3-Data-Visualization/12-visualization-relationships/README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@
 
 Continuing with the nature focus of our research, let's discover interesting visualizations to show the relationships between various types of honey, according to a dataset derived from the [United States Department of Agriculture](https://www.nass.usda.gov/About_NASS/index.php). 
 
-This dataset of about 600 items displays honey production in many U.S. states. So, for example, you can look at the number of colonies, yield per colony, total production, stocks, price per pound, and value of the honey produced in a given state from 1998-2012, with one row per year for each state. 
+This dataset of about 600 items, displays honey production in many U.S. states. So, for example, you can look at the number of colonies, yield per colony, total production, stocks, price per pound, and value of the honey produced in a given state from 1998-2012, with one row per year for each state.
```
Comment (jlooper) [inline 3-Data-Visualization/12-visualization-relationships/README.md:9]: hi, please roll this back as we don't need the extra comma here

File: 3-Data-Visualization/12-visualization-relationships/README.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ ax.figure.legend();
 ```
 ![superimposed plots](images/dual-line.png)
 
-While nothing jumps out to the eye around the year 2003, it does allow us to end this lesson on a little happier note: while there are overall a declining number of colonies, their numbers might seem to be stabilizing and their yield per colony is actually increasing, even with fewer bees. 
+While nothing jumps out to the eye around the year 2003, it does allow us to end this lesson on a little happier note: while there are overall a declining number of colonies, their numbers might seem to be stabilizing.
```
Comment (jlooper) [inline 3-Data-Visualization/12-visualization-relationships/README.md:159]: why did you remove the last sentence? Let's add it back in, please.

**Review by jlooper — CHANGES_REQUESTED** [review]

File: 3-Data-Visualization/12-visualization-relationships/README.md (hunk 2 above)
Comment (iofall) [reply 3-Data-Visualization/12-visualization-relationships/README.md:159]: This line was changed because, if we look at the plot for `yield per colony vs year`, the `yield per colony` is actually decreasing over the years.

image

Comment (jlooper) [reply 3-Data-Visualization/12-visualization-relationships/README.md:159]: ah, thank you, I see it. The red line does show the yield going down. Maybe we should say that "the number of colonies is stabilizing even if their yield per colony is decreasing."

Comment (iofall) [reply 3-Data-Visualization/12-visualization-relationships/README.md:159]: Got it 👍

Comment (jlooper) [reply 3-Data-Visualization/12-visualization-relationships/README.md:159]: thanks much!

File: 3-Data-Visualization/12-visualization-relationships/README.md [hunk 3]
```diff
@@ -156,7 +156,7 @@ ax.figure.legend();
 ```
 ![superimposed plots](images/dual-line.png)
 
-While nothing jumps out to the eye around the year 2003, it does allow us to end this lesson on a little happier note: while there are overall a declining number of colonies, their numbers might seem to be stabilizing and their yield per colony is actually increasing, even with fewer bees. 
+While nothing jumps out to the eye around the year 2003, it does allow us to end this lesson on a little happier note: while there are overall a declining number of colonies, the number of colonies is stabilizing even if their yield per colony is decreasing.
```
Comment (iofall) [inline 3-Data-Visualization/12-visualization-relationships/README.md:159]: I do wonder though, if we look at the plot, the `number of colonies` haven't changed much. Is it justified to say '_there are overall a declining number of colonies_'? Or was that meant as a holistic, outside of the dataset / general statement.

Comment (jlooper) [reply 3-Data-Visualization/12-visualization-relationships/README.md:159]: yes, I think it was a holistic statement. It's not the greatest sentence in the world :D - let me commit this for now for launch but we can look again shortly
